.prettierrc

```
@@ -0,0 +1,6 @@
+bracketSpacing: true
+trailingComma: 'all'
+useTabs: false
+tabWidth: 4
+semi: true
+singleQuote: true
```

`package-lock.json` is the most changed file with +20,000 changes; if you don't want to change much at all, I would recommend to set `tabWidth: 2` for all json files.

```suggestion
singleQuote: true
overrides:
    - files: '*.json'
      options:
          tabWidth: 2
```